core/src/jvmTest/kotlin/com/ustadmobile/core/contentjob/TestContentJobRunner.kt

```diff
@@ -0,0 +1,201 @@
+package com.ustadmobile.core.contentjob
+
+import com.ustadmobile.core.account.Endpoint
+import com.ustadmobile.core.account.UstadAccountManager
+import com.ustadmobile.core.db.JobStatus
+import com.ustadmobile.core.db.UmAppDatabase
+import com.ustadmobile.core.networkmanager.ConnectivityLiveData
+import com.ustadmobile.core.util.UstadTestRule
+import com.ustadmobile.core.util.directActiveDbInstance
+import com.ustadmobile.core.util.onActiveAccount
+import com.ustadmobile.door.DoorUri
+import com.ustadmobile.door.ext.DoorTag
+import com.ustadmobile.lib.db.entities.*
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import org.junit.Rule
+import kotlin.test.Test
+import org.junit.Before
+import org.kodein.di.*
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.junit.Assert
+import org.mockito.kotlin.stub
+
+class TestContentJobRunner {
```

Comment:
This needs a test to check the cancellation of a job item due to unacceptable connectivity. That is an important job of the TestContentjobRunner, but it is not covered by these tests.

e.g. givenContentJobItemRunning_whenConnectivityBecomesUnacceptable_thenShouldCancel

You could create another dummy plugin that simply executes a delay function, and then that should be canceled.